### PR TITLE
Add and update some notes in tutorials

### PR DESCRIPTION
**Pull request by AlaricChan**
**Background**
The changes aim to optimize the tutorials, make it easier and more smooth to understand for newbies.
1. Update some notes.
2. Correct words, remove useless blanks in code.

**Details**
1. Since the OBM setting is a core configuration in several features,  update the note in sector of configuring OBM settings, make it more clear about what should be noticed to ensure the settings is configured successfully.
2. Add notice in paragraph of Install OS with RackHD API. The original repo address is lowercase in the downloaded payload json, but folder name of mirror is configured with a upper character in previous step. Such case would cause a mismatch easily, lead to the failure of OS installation workflow.

**Reviewer**
@iceiilin @anhou @yaolingling

### CODE REVIEW

**Comment by leoyjchang** [comment]
+1 to me, thanks @AlaricChan

File: docs/tutorials/vagrant-installos.rst [hunk 1]
```diff
@@ -55,15 +55,15 @@ For development environment,
...
-   
+
      vagrant@rackhd:~$ sudo su
      root@rackhd:/home/vagrant# mkdir iso && cd iso
      root@rackhd:/home/vagrant# mkdir -p /var/mirrors/Photon
      root@rackhd:/home/vagrant# mkdir -p /var/renasar/on-http/static/http/mirrors/
-   
-2. Create OS mirror from an ISO image by typing below command. (Note: The photon-1.0-13c08b6.iso needs to be downloaded in /home/vagrant/iso)
+
+2. Create OS mirror from an ISO image by typing below command. (Note:The photon-1.0-13c08b6.iso needs to be downloaded in /home/vagrant/iso. Search the download URL, download it with `wget` command.)
```
Comment (yaolingling) [inline docs/tutorials/vagrant-installos.rst]: write a specific download link.

Comment (iceiilin) [reply docs/tutorials/vagrant-installos.rst]: agree

**Comment by AlaricChan** [comment]
Closed by mistake. Reopen for review. Thank you guys.
@iceiilin @yaolingling @leoyjchang @anhou

**Pull request merged**